chart/values.yaml

```diff
@@ -313,3 +279,75 @@ web:
 
   ## @param web.affinity Affinity to add to the web pod
   affinity: {}
+
+### @section XNAT Database parameters
+### @param postgresql.enabled Enable or disable the PostgreSQL deployment
+### @param postgresql.auth.database PostgreSQL database name
+### @param postgresql.auth.username PostgreSQL username
+### @param postgresql.auth.password PostgreSQL password. Make sure to override this.
+### @param postgresql.auth.postgresPassword PostgreSQL password. Make sure to override this.
+### @param postgresql.image.tag PostgreSQL image tag
+### @param postgresql.primary.resources.requests.cpu CPU request
+### @param postgresql.primary.resources.requests.memory Memory request
+### @param postgresql.primary.resources.limits.cpu CPU limit
+### @param postgresql.primary.resources.limits.memory Memory limit
+#postgresql:
+#  enabled: true
+#  auth:
+#    database: xnat
+#    username: xnat
+#    password: xnat
+#    postgresPassword: postgres
+#  image:
+#    tag: 14.17.0-debian-12-r2
+#  primary:
+#    resources:
+#      requests:
+#        cpu: 1
+#        memory: 4000Mi
+#      limits:
+#        cpu: 2
+#        memory: 4000Mi
+#
+
+### @section XNAT Database parameters
+## @param postgresql.enabled Whether to deploy a PostgreSQL cluster
+## @param postgresql.backups.enabled Whether to enable database backups
+## @param postgresql.cluster.imageName Name of the PostgreSQL container image
+## @param postgresql.cluster.instances Number of PostgreSQL instances
+## @param postgresql.cluster.postgresql.parameters.shared_buffers Amount of memory used for shared buffers
+## @param postgresql.cluster.resources.requests.cpu CPU request
+## @param postgresql.cluster.resources.requests.memory Memory request
+## @param postgresql.cluster.resources.limits.cpu CPU limit
+## @param postgresql.cluster.resources.limits.memory Memory limit
+## @param postgresql.cluster.storage.size Size of the storage
+## @param postgresql.cluster.initdb.database PostgreSQL database name
+## @param postgresql.cluster.initdb.owner PostgreSQL owner
+## @param postgresql.cluster.initdb.secret.name Name of the secret containing credentials for the database
+## @param postgresql.cluster.version.postgresql PostgreSQL major version to use
+postgresql:
+  enabled: true
+  backups:
+    enabled: false
+  cluster:
+    imageName: ghcr.io/cloudnative-pg/postgresql:14.17-standard-bookworm
+    instances: 3
+    postgresql:
+      parameters:
+        shared_buffers: "512MB"
```

Comment:
25% of the requested memory (recommended in the CNPG docs, see: https://www.postgresql.org/docs/14/runtime-config-resource.html#GUC-SHARED-BUFFERS)